### PR TITLE
Adapt the README after the code was moved to core and maven separate repos

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling could you please review this one together with https://github.com/srcdeps/srcdeps-core/pull/5 and https://github.com/srcdeps/srcdeps-maven/pull/5 It is just a basic cleanup after the split of the repos. The new functionalty is not covered yet - it is still on my TODO.

**Comment by ppalaga** [comment]
Here "cleanup" means cleanup of the READMEs so that we do not have three identical READMEs.

File: README.adoc [hunk 1]
```diff
@@ -1,106 +1,19 @@
...
-Originally, the present project provided a Maven plugin called `srcdeps-maven-plugin` that worked fairly well in most situations, but it turned out soon that the Maven Plugin API is not rich enough to allow for catching dependencies of
-all possible kinds. Most notably, the given project's parent pom and imports in `dependencyManagement` sections were
-resolved by Maven even before the plugin was initialized. Thus, if the given project's parent pom had a `-SRC-`
-version, the build failed with a missing artifact error message, simply because it happened before
-`srcdeps-maven-plugin` could do anything.
 
-An outdated presentation about the abandoned `srcdeps-maven-plugin`: http://ppalaga.github.io/presentations/150929-srcdeps-maven-plugin/150929-srcdeps-maven-plugin.html
+== Srcdeps support in build tools
 
+Srcdeps is fully supported in in Maven 3.3.1+. Please refer to
```
Comment (jpkrohling) [inline README.adoc]: `s/in in/in`

File: README.adoc [hunk 2]
```diff
@@ -1,106 +1,19 @@
...
+link:https://github.com/srcdeps/srcdeps-maven[Srcdpes Maven] for details for to configure etc.
```
Comment (jpkrohling) [inline README.adoc]: `s/Srcdpes/Srcdeps/`

**Review by jpkrohling — COMMENTED** [review]
Apart from a couple of typos, I think it's OK. I cannot judge, though, whether everything that has been removed here was previously added to the destination :)

**Pull request merged**